support-frontend/assets/helpers/storage/abandonedBasketCookies.ts

```diff
@@ -26,7 +25,7 @@ const abandonedBasketSchema = object({
 type AbandonedBasket = InferInput<typeof abandonedBasketSchema>;
 
 export function useAbandonedBasketCookie(
-	product: ProductCheckout,
+	product: string,
```

Comment:
ideally I would make this more type safe instead of less, but I couldn't figure out how to simply make the types match

Reply:
We are playing a little fast and loose here. At the moment this is `string` - as seen in [`productCatalog`](https://github.com/guardian/support-frontend/blob/main/support-frontend/assets/helpers/productCatalog.tsx#L22).

If we are going to force it we should probably do it there.

Saying that, I just thought, maybe to avoid having TODOs all over the shot, we could add something like [this](https://github.com/guardian/support-frontend/pull/6133)?

Reply:
that's a nice idea! Still a little bit of confusion about whether it has names like eg `NationalDelivery` or just `Paper`. Which we've included in [SDC](https://github.com/guardian/support-dotcom-components/blob/b84199a728184b11dcc7fbeb8de482cd996edb28/packages/shared/src/types/targeting/shared.ts#L47) and could cause issues further down the line

Reply:
I think given that the source is our aimed solution for these things - we should try to keep the `ProductCatalogApi` as the source for the types. 

I am going to have a go at publishing the types to NPM at `@guardian/product-catalog`.

It will require syncing everything up but should avoid this model proliferation we have.